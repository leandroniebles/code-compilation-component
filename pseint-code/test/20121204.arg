--force_init_vars=1 --input=4 --input=1 --input=2 --input=3 --input=4