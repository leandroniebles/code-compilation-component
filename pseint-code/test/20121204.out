Ingrese la cantidad de datos:
> 4
Ingrese el dato 1:
> 1
Ingrese el dato 2:
> 2
Ingrese el dato 3:
> 3
Ingrese el dato 4:
> 4
El promedio es: 2.5
